.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**[Optional]**

- [ ] Kindly add a license file in your project because you have a link to your license file in your readme file. You can add a license file using these [guildines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: